File: Python/Code/2500/2588 - 곱셈.py
a = input()
b = input()
a = int(a)
b = int(b)
result = 0
temp = a * (b % 10)
result += temp
print(temp)
b //= 10
temp = a * (b % 10)
result += (temp * 10)
print(temp)
b //= 10
temp = a * b
result += (temp * 100)
print(temp)
print(result)

File: Java/Code/2500/2588 - 곱셈.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int r = 0, a, b, digit = 1;
		a = sc.nextInt();
		b = sc.nextInt();
		while (b > 0) {
			int temp = b % 10, tempResult = temp * a;
			System.out.println(tempResult);
			r += (tempResult * digit);
			digit *= 10;
			b /= 10;
		}
		System.out.println(r);
	}
}
